### PR TITLE
Adding support for H1 FLR messages

### PR DESCRIPTION
**Pull request by makrsmark**
same format as WRN

### CODE REVIEW

File: lib/plugins/Label_H1_FLR.test.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  expect(decodeResult.decoder.decodeLevel).toBe('partial');
+  expect(decodeResult.decoder.name).toBe('label-h1-flr');
+  expect(decodeResult.formatted.description).toBe('Fault Log Report');
+  expect(decodeResult.message.text).toBe(text);
+  expect(decodeResult.raw.message_timestamp).toBe(1712138580);
+  expect(decodeResult.formatted.items.length).toBe(1);
+  expect(decodeResult.formatted.items[0].type).toBe('fault');
+  expect(decodeResult.formatted.items[0].code).toBe('FR');
+  expect(decodeResult.formatted.items[0].label).toBe('Fault Report');
+  expect(decodeResult.formatted.items[0].value).toBe('LIQD LVL XMTR (40MG)/IDTOILET ');
```
Comment (kevinelliott) [inline lib/plugins/Label_H1_FLR.test.ts:105]: Seems like we could further parse this down.

**Review by kevinelliott — APPROVED** [review]
I think the faults could be further parsed down, for example you can see evidence of the fault identification (TOILET), etc. But, I don't want to block merge for this as long as we think about cycling back to refine later.

**Comment by makrsmark** [comment]
@kevinelliott I definitely agree it can be parsed down further. I just don't have enough data to really see what the format is. I'm guessing something like `type subtype data thing` but I'm not confident

**Comment by makrsmark** [comment]
I forgot to mention, i believe FLR and WRN share the same message format

**Pull request merged**